utils/src/nativeCommonMain/kotlin/com/badoo/reaktive/utils/Freeze.kt

```diff
@@ -4,12 +4,13 @@ import kotlin.native.concurrent.ensureNeverFrozen
 import kotlin.native.concurrent.freeze
 import kotlin.native.concurrent.isFrozen
 
-actual fun <T> T.freeze(): T = freeze()
+@Suppress("NOTHING_TO_INLINE")
+actual inline fun <T> T.freeze(): T = freeze()
 
 actual fun <T: Any> T.ensureNeverFrozen(): T {
```

Comment:
Probably this one is worth becoming `inline`d too

Reply:
Yeah, let's do it. Also inline `isFrozen` please.

Reply:
And better to annotate `expect` functions as inline as well

Reply:
`isFrozen` is already inlined. Agree that need to move modifier to `expect` functions, thought about after pushing the PR too